frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -147,6 +150,17 @@ const HostSummary = ({
             id={"disk-space-tooltip"}
           />
         </div>
+        {typeof diskEncryption?.enabled === "boolean" &&
```

Comment:
is this the correct check? Here we are checking that `diskEncryption.enabled` is a boolean type, but isn't what we need to check is if `diskEncryption.enabled` is `true`?

If `diskEncryption.enabled` is a boolean it can still be `false`, and imagine we don't wanna show this UI when false.

Reply:
I had the same question. There's a requirement where the value can be "null", in which case don't show anything. If it's "true" or "false", then show it with the appropriate value.

Reply:
Per specs, `disk_encryption_enabled` from the API can return null for Linux hosts since there's no way of knowing if it's disabled. If that's the case, the spec is to not render any disk encryption.

Is there a better way for checking that it's a boolean to render this disk encryption section?

Reply:
ok makes sense. this looks good then